chatops/test-hubot-st2-badconfig.bats

```diff
@@ -0,0 +1,105 @@
+
+load '../test_helpers/bats-support/load'
+load '../test_helpers/bats-assert/load'
+
+
+ST2_CHATOPS_ENV_FILE='/opt/stackstorm/chatops/st2chatops.env'
+
+@test "SETUP: setup environment variables for authentication related variables." {
+	`sed -e '/ ST2_API_KEY=/s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_TOKEN= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_USERNAME= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_PASSWORD= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`
+	`sed -e '/ ST2_AUTH_URL= /s/^/#/g' -i $ST2_CHATOPS_ENV_FILE`;
+}
+
+@test "exit with error status from unresolved ST2_API_URL." {
+	run eval "("\
```

Comment:
Have `eval` because check for return status and make sure its value is 1 `assert_equal $status 1`
Maybe there are better way to do it.  Will do more investigations and clean up unnecessary new line.

Reply:
Yes, see https://github.com/ztombol/bats-assert
Please try `run` without `eval` and see if that works.

I think it could be addressed by https://github.com/StackStorm/st2tests/pull/170/files#r288962412

Reply:
@armab Without `eval`,  it can't handle environment variable passed to `st2chatops`
```✗ exit with error status from unresolved ST2_API_URL.
   (from function `assert_failure' in file st2tests/chatops/../test_helpers/bats-assert/src/assert.bash, line 149,
    in test file st2tests/chatops/test-hubot-st2-badconfig.bats, line 20)
     `assert_failure 1' failed
     /usr/lib/bats/bats-exec-test: line 58: ST2_API_URL=http://non-existent:9101/: No such file or directory

   -- command failed as expected, but status differs --
   expected : 1
   actual   : 127
   output   : /usr/lib/bats/bats-exec-test: line 58: ST2_API_URL=http://non-existent:9101/: No such file or directory
   --```